modules/ROOT/partials/rtc/rtc-client-connect-inputs.adoc

```diff
@@ -4,7 +4,7 @@
 
 | `userId`
 | `string`
-| This is the user's unique ID (the `sub` field from their link:({baseurl}/rtc/jwt-authentication/)[JWT], which is also used for <<rtc_user_details_provider,`rtc_user_details_provider`>>). Multiple connection events will be received with the same user ID if a user opens multiple sessions (for example on desktop and mobile).
+| This is the user's unique ID (the `sub` field from their xref:rtc/jwt-authentication.adoc[JWT], which is also used for <<rtc_user_details_provider,`rtc_user_details_provider`>>). Multiple connection events will be received with the same user ID if a user opens multiple sessions (for example on desktop and mobile).
```

Comment:
There's a legacy xref here that needs converting.

Reply:
done, but it seems not to work on this page: https://www.tiny.cloud/docs/rtc/events/#rtc_client_info
both in production and in local

Reply:
You're correct there are 3 broken xrefs/links on this page. There should be something like this:
```
xref:rtc/configuration/rtc-options-optional.adoc#rtc_user_details_provider[`rtc_user_details_provider`]
```
and
```
xref:rtc/configuration/rtc-options-optional.adoc##rtc_client_info[`rtc_client_info`]
```

Reply:
done